static/src/javascripts/projects/common/modules/commercial/dfp/render-advert.js

```diff
@@ -48,8 +48,8 @@ define([
      */
     sizeCallbacks[adSizes.mpu] = function (_, advert) {
         if (advert.node.classList.contains('ad-slot--right')) {
-            var mobileAdSizes = advert.sizes('data-mobile');
-            if (mobileAdSizes && mobileAdSizes.indexOf([300, 251]) > -1) {
+            var mobileAdSizes = advert.sizes.mobile;
+            if (mobileAdSizes && mobileAdSizes.some(function (size) { return size[0] === 300 && size[1] === 251; })) {
```

Comment:
This function reads a bit implicitly because of the lack of structured typing, which appears to come from `dfp-api`, in the local function `createSizeMapping`. Maybe if `createSizeMapping` returned an object with `width` and `height`, the code would be more readable and debuggable? Not a big issue.